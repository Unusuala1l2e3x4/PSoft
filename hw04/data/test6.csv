"char1","book1"
"char1","book2"
"char1","book3"
"char2","book1"
"char2","book2"
"char2","book3"
"char3","book1"
"char3","book2"
"char3","book3"
