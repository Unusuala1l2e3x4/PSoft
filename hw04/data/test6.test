r
93
143
q