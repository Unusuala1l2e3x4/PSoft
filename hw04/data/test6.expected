First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [93]
Unknown building: [143]
